maturity-level-one/src/Controllers/PlayersController.cs

```diff
@@ -100,7 +96,7 @@ public async Task<IActionResult> UpdateFull(int id, [FromBody] PlayerDto player)
         }
 
         [HttpPatch("{id}")]
-        [SwaggerOperation("update-player-incremental")]
+        [SwaggerOperation("Players_UpdateIncremental")]
         [SwaggerResponse(204, "No Content")]
         [SwaggerResponse(404, "Player not found")]
         [SwaggerResponse(500, "API is not available")]
```

Comment:
@MassimoC 	Did you implement this? If so, shouldn't this be a patch and UpdateFull be renamed to Update?

Reply:
Taking this offline with Massimo.